src/main/java/com/kdt/localinfo/model/User.java

```diff
@@ -0,0 +1,89 @@
+package com.kdt.localinfo.model;
+
+import com.kdt.localinfo.controller.user.UserRequest;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@NoArgsConstructor
+@Getter
+@Entity
+@AllArgsConstructor
+@Builder
+public class User {
```

Comment:
테이블 이름 user로 생성되어서 users로 바꾸면 좋을 것 같습니다.
```suggestion
@Table(name = "users")
public class User {
```